### PR TITLE
Test link

### PR DESCRIPTION
**Pull request by maektwain**
##### Description

It aims to provide the conditional message showing with link 
##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)

No
##### Testing

Yes
##### Refers/Fixes

Fixes: #79

### CODE REVIEW

File: src/components/message.js [hunk 1]
```diff
@@ -26,15 +26,27 @@ function showMessage(type, content, duration) {
...
-  msgHandlers[msgKey] = m;
+  const msgBodywithLink = (
+    <span>
+      {content.message} <a href={content.link}> View on Etherscan</a>
+        <Icon type="close-circle" onClick={handleDismiss} style={iconStyle} />
+      </span>
+  );
+
+
+  if (content.link){
```
Comment (eswarasai) [inline src/components/message.js]: Fix the formatting issues here please

File: src/components/message.js [hunk 2]
```diff
@@ -26,15 +26,27 @@ function showMessage(type, content, duration) {
...
+    let  m = message[type](msgBodywithLink, duration, handleClose);
```
Comment (eswarasai) [inline src/components/message.js]: Define `let m` above if block and then assign it inside the `if` block

Comment (maektwain) [reply src/components/message.js]: Cool

File: src/components/message.js (hunk 1 above)
Comment (maektwain) [reply src/components/message.js]: Cool

**Comment by pelsasser** [comment]
@maektwain thanks for the PR! @eswarasai good to merge.  The link is a little hard to read, but we can address that at a later date or in another issue.  I will open that as low priority issue now.

image

**Pull request merged**